spec/fixtures/shelter_1.json

```diff
@@ -0,0 +1,14 @@
+{
+  "data": {
+  "id": "1",
+  "type": "shelter",
+  "attributes": {
+  "name": "red barn"
+  },
+  "relationships": {
+  "animals": {
+  "data": []
+  }
+  }
+  }
+  }
```

Comment:
could we fix the indentations here? Is this right?
```
{
  "data": {
    "id": "1",
    "type": "shelter",
    "attributes": {
      "name": "red barn"
    },
    "relationships": {
      "animals": {
        "data": []
      }
    }
  }
}
```

Reply:
That looks correct. That fixture was temporary (just pulled from seed data on my back-end's local), just to get something to show it was working. Once a real fixture is in place, the tests will use that.